Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -1046,6 +1046,45 @@ struct molops_wrapper {
                 docString.c_str(),
                 python::return_value_policy<python::manage_new_object>());
 
+    // ------------------------------------------------------------------------
+    docString = R"DOC(Parameters controlling which Hs are removed.)DOC";
+    python::class_<MolOps::RemoveHsParameters>("RemoveHsParameters",
+                                               docString.c_str())
+        .def_readwrite("removeDegreeZero",
```

Comment:
I would copy the comments from the struct into the doctoring.  I'm unsure if readwrite can have doc strings, but if so that would be good as well.